docs/Spark/fabrics/emr-fabric-serverless.md

```diff
@@ -0,0 +1,103 @@
+
+1. Modify the **Primary Node** security group:
+   - Allow incoming connections to port `8998` from the Prophecy IP.
+   - This can be achieved by adding an inbound rule to the Master security group that permits incoming traffic on port `8998` from the Prophecy IP address.
+2. Modify the **Core Node** security group:
+   - Allow outgoing connections to the Prophecy public IP over HTTPS.
+   - To enable this, add an outbound rule to the Core security group that allows outgoing traffic over HTTPS protocol to the Prophecy public IP.
+
+By implementing these adjustments to your EMR Serverless cluster's security groups, you will establish the necessary network configuration for effective integration with Prophecy.
+
+## Create a Fabric to connect Prophecy to EMR Serverless
+
+Navigate to Prophecy's UI and click on **Create Fabric**. The Fabric will establish a connection with your EMR Serverless cluster and use it as the execution engine for your Pipelines.
+<img src={require('./img/create-entity-emr.png').default} alt="EMR Serverless create Fabric" width="75%" />
+<br/>
+<br/>
+
+**Name** your EMR Serverless Fabric and click on **Continue**.
+
+<br/>
+<br/>
+
+<img src={require('./img/name-emr-serverless.png').default} alt="EMR Serverless Fabric name" width="80%" />
+
+Choose **EMR** as your **Provider**. At this point, there is no distinction between creating an EMR and an EMR Serverless Fabric.
+
+<img src={require('./img/emr-f-abric.png').default} alt="EMR Serverless Provider" width="75%" />
+
+Before proceeding, it is crucial to ensure that all the required settings are properly configured. If you are uncertain about the necessary configurations, we recommend reaching out to your cloud infrastructure team for additional guidance and assistance. They will be able to provide you with the specific information and support needed to ensure a successful setup.
+
+Enter your AWS credentials under **Access Key** and **Secret Key**. Choose the **Region** that your EMR Serverless cluster is running in.
+
+Click on **Fetch environments**.
+<img src={require('./img/emr-aws-cred.png').default} alt="EMR Serverless cred" width="75%" />
+
+Select your EMR Serverless cluster that you would like to connect to under **Spark Environment**. The EMR cluster listing also shows Serverless clusters. From the list of active clusters, you can identify your EMR Serverless cluster by finding `emr-serverless-services` within the Livy URL.
+
+<img src={require('./img/emr-serverless-dev-select.png').default} alt="EMR Serverless select" width="80%" />
+
+Most of the fields should be automatically populated after selecting a EMR Serverless cluster.
+
+Optional: You can enable Lake Formation by clicking the toggle.
+
+Select your Authentication type. We suggest choosing `AWS Sig V4`.
+<img src={require('./img/emr-serverless-auth-type.png').default} alt="EMR Serverless Authentication type" width="50%" />
```

Comment:
For EMR serverless, this is the only auth that will work. So, can we use a stronger word than "suggest"?

Reply:
Done.